### ISSUE TITLE
Use phone local time for Topic Updates

### ISSUE THREAD

**Issue by sgvj** [dc297/mqttclpro#24]
The times listed by the topics and the times listed for messages within each topic are 7 hours ahead.
I have verified that my phone timezone is set correctly etc..
ps
Thanks for the awesome app. This new tasker-subscribe feature has made my week/year.

**Comment by acefed** [comment]
I think the app is time zone agnostic, and shows times in UTC/GMT. I'm in CET (GMT+2, with daylight saving) and the app shows my times -2 hours.

**Comment by dc297** [comment]
Hey, I have found the issue. Should be fixed in the next version.

Cheers
Deepesh

**Comment by sgvj** [comment]
Thanks, enjoy a beer!
Also, how about adding a donate button to the project page to avoid that 30%?

**Comment by dc297** [comment]
Hey Thanx for the beer.

I have pushed the latest update to Play store. About that donate button, I don't know why but PayPal has disabled sending/receiving money to/from India. So I am only left with Google Play's In app purchases.

Cheers!

**Comment by acefed** [comment]
Tested the time zone change: Works like a charm now!

Received messages (especially for time based events, like sunrise for example) make so much more sense now. Loving this change even more than expected :)

----------------------
I looked into PayPal India a bit, but found mostly old information. The government closed in on PayPal, because of tax evasion reasons. Those using it need(ed) a merchant account, can't store money on PayPal, and had to transfer earnings to their Indian bank within 7 days. I saw fees around 4% for that service.
It might be wise to look into further, 30% vs <5% is a huge difference if you grow big. And you will :)
